hack/run_test

```diff
@@ -27,6 +24,13 @@ function cleanup() {
     docker rm $CONTAINER
     rm $cidfile
     echo "Done."
+}
+
+function cleanup() {
+  for cidfile in $CIDFILE_DIR/* ; do
+    CONTAINER=$(cat $cidfile)
+
+    cleanup_container
```

Comment:
I would move this into the second patch, otherwise this "function split" looks like
unnecessary (when somebody looks at the patch separately).